### DISCUSSION TITLE
need help with CMD commands

### DISCUSSION THREAD

**Discussion by DiegoFiorellino** [MRColorR/money4band#65] [Q&A] [answered]
Guys, I need help, when in the CMD I go to put the commands, nothing happens, could someone help me? Thank you very much

**Comment by MRColorR** [comment]
Hello,
Could you please tell us what are the steps you are taking. 
Also on which OS are you and are you using the latest release or pre-release?
Thank you

**Reply by DiegoFiorellino** [reply]
Hi, I'm stuck at step 3 where I have to put the commands in the promt of the commands but when I copy and paste them into the cmd, it tells me this: "Set-ExecutionPolicy" is not recognized as an internal or external command,
  an executable program or batch file. I'm using Windows 10 and I'm using the latest version. Thank you very much.

**Reply by MRColorR** [reply, marked as answer]
Got it, thanks for the details: you should use windows powershell, as cmd is older and has less functionalities( it cannot understand Powershell specific commands). Powershell 5 should be already present in windows 10 and windows 11 and it should be enough to run the scripts , I've tested them on ps7 tough. 

Just search Powershell in windows search bar, then right click and open as administrator , navigate to the money4band folder using CD and then start the ps1 script. 
Let me know if you need further assistance 🙂
